server/api/services/events.service.js

```diff
@@ -156,17 +157,19 @@ class EventsService {
         points[`score.${eventData.category}`] = eventData.importance * 5 + (2 - index) * 5;
         points['totalScore'] = eventData.importance * 5 + (2 - index) * 5;
 
-        const achievement = {};
-        if (index === 0) achievement['achievements.first'] = eventData.name;
-        else if (index === 1) achievement['achievements.second'] = eventData.name;
-        else if (index === 2) achievement['achievements.third'] = eventData.name;
-
+        winner.forEach(item => {
+          const newAchievement = new Achievement({
```

Comment:
Instead of creating an instance of the Achievement model and then saving it, we can rather use the `.create()` function like we did at other places in the codebase. You can then add them to the `promises` array which is already present.